drivers/gpio/gpio_intel_apl.c

```diff
@@ -265,7 +265,7 @@ static int gpio_intel_apl_config(struct device *dev, int access_op,
 	/* setup interrupt if desired */
 	if (flags & GPIO_INT) {
 		/* invert signal for interrupt controller */
-		if ((flags & GPIO_INT_ACTIVE_HIGH) == GPIO_INT_ACTIVE_LOW) {
+		if ((flags & GPIO_INT_ACTIVE_HIGH) != 0) {
```

Comment:
I believe the sense of this is wrong, since `GPIO_INT_ACTIVE_LOW`  is zero.

Reply:
Seems like this should be `if ((flags & GPIO_INT_ACTIVE_HIGH) == 0)`

Reply:
Fixed. Sorry!